### PR TITLE
media-radio/KochMorse: Updated DEPEND, RDEPEND

### PR DESCRIPTION
**Pull request by rseichter**
Closes: https://bugs.gentoo.org/684890

### CODE REVIEW

File: media-radio/KochMorse/KochMorse-3.4.0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+SRC_URI="https://github.com/hmatuschek/kochmorse/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtmultimedia
```
Comment (a17r) [inline media-radio/KochMorse/KochMorse-3.4.0-r1.ebuild]: missing slot

File: media-radio/KochMorse/KochMorse-3.4.0-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	dev-qt/qtwidgets:5"
+DEPEND="${RDEPEND}
+	dev-qt/linguist-tools"
```
Comment (a17r) [inline media-radio/KochMorse/KochMorse-3.4.0-r1.ebuild]: 1) missing slot
2) in EAPI-7, this is in `BDEPEND` instead

File: media-radio/KochMorse/KochMorse-3.4.0-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (a17r) [inline media-radio/KochMorse/KochMorse-3.4.0-r1.ebuild]: EAPI-7 please

File: media-radio/KochMorse/KochMorse-3.4.0-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+DEPEND="${RDEPEND}
+	dev-qt/linguist-tools"
+
+S="${WORKDIR}/kochmorse-${PV}"
+
+src_prepare() {
+	# Upstream uses a non-standard category in release 3.4.0. I submitted
+	# a fix which has been accepted, but not yet released.
+	sed -i -e 's/Teaching;/X-Teaching;/' shared/kochmorse.desktop || die
+	eapply_user
```
Comment (a17r) [inline media-radio/KochMorse/KochMorse-3.4.0-r1.ebuild]: missing `cmake-utils_src_prepare`, call it before anything else in src_prepare and drop `eapply_user`

File: media-radio/KochMorse/KochMorse-3.4.0-r1.ebuild (hunk 2 above)
Comment (rseichter) [reply media-radio/KochMorse/KochMorse-3.4.0-r1.ebuild]: I'll add the slot. As for `BDEPEND`, the build currently uses EAPI 6, and I don't feel confident changing that given that I cannot build myself.

**Comment by rseichter** [comment]
@a17r @pvdabeel : Did somebody just hijack my pull request? I now see an indication that there are conflicts.

**Comment by rseichter** [comment]
Hmm, now the PR is closed. No idea what happened before, but it sure looked odd to me.

**Comment by a17r** [comment]
Just GitHub doing its thing while gentoo-bot was kept busy closing PRs.

**Pull request closed without merging**